README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider adding a note regarding the compatibility of 'go-task' with the installed version of Go. It would be helpful for the user to ensure they have the right version of Go installed before proceeding with 'go-task'.